apps/omg_eth/config/dev.exs

```diff
@@ -1 +1,5 @@
 use Mix.Config
+
+# bumping because otherwise import/unlock of accounts will fail in some of use cases (perf tests)
+config :ethereumex,
+  request_timeout: 60_000
```

Comment:
Maybe it's worth looking to ethereumex 0.5.1? Looks like they added those timeouts (from their github):
```
config :ethereumex,
  http_options: [timeout: 8000, recv_timeout: 5000]
```

Reply:
ok, let's do grooming of ethereumex dep in a different task/chore